pkg/types/task.go

```diff
@@ -108,10 +108,14 @@ var DefaultTaskPolicy = TaskPolicy{
 	Timeout:    3600,
 }
 
+var MaxTaskExpirationS = 24 * 60 * 60
+var MaxTaskRetries = 5
+
 type TaskPolicy struct {
 	MaxRetries uint      `json:"max_retries" redis:"max_retries"`
 	Timeout    int       `json:"timeout" redis:"timeout"`
 	Expires    time.Time `json:"expires" redis:"expires"`
```

Comment:
I think it's confusing that we're adding an additional field here on task policy, expires and expiration sound the same to me. I know that they represent different things but on the go side I wouldn't know that they did